dev-tools/jenkins_ci.sh

```diff
@@ -159,9 +159,9 @@ case `uname` in
         exit 4
         ;;
 esac
-
-# If this isn't a PR build and isn't a debug build then upload the artifacts
-if [[ -z "$PR_AUTHOR" && -z "$ML_DEBUG" ]] ; then
-    (cd .. && ./gradlew --info -Dbuild.version_qualifier=$VERSION_QUALIFIER -Dbuild.snapshot=$BUILD_SNAPSHOT upload)
-fi
+# Do not upload artifacts from the Jenkins' build while testing uploading from BuildKite pipelines
```

Comment:
Something similar needs doing in `dev-tools/jenkins_ci.ps1`.

And as a result of this change one more change will also be needed to the Jenkins builds.

Please add an `exit 0` and a comment saying why near the beginning of `dev-tools/jenkins_combine_artifacts.sh`. This won't matter for PR builds, but will stop the Jenkins snapshot and staging builds from failing due to missing artifacts, or from fighting with the BuildKite builds.